packages/itwinui-css/src/bridge.scss

```diff
@@ -195,7 +195,7 @@
 
   .iui-checkbox-wrapper {
     &:not(:where([data-iui-status])):not([data-iui-disabled='true']) {
-      // Kiwi labels are slightly muted
+      // IDS labels are slightly muted
```

Comment:
It's unclear if "IDS" will be used to refer to the system as a whole (including v3) or just the new parts. We can use "v5" or "iTwinUI v5" for now until we have more clarity.